jdbc/spring-cloud-starter-stream-source-jdbc/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1,3 +1,4 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.jdbc.source.JdbcSourceProperties, \
-  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceProperties
```

Comment:
@artembilan @ericbottard This is what I meant by my first comment:\

```
dataflow:>app info source:jdbc
Information about source application 'jdbc':
Resource URI: maven://org.springframework.cloud.stream.app:jdbc-source-rabbit:1.0.0.BUILD-SNAPSHOT
╔══════════════════════════════╤══════════════════════════════╤══════════════════════════════╤══════════════════════════════╗
║         Option Name          │         Description          │           Default            │             Type             ║
╠══════════════════════════════╪══════════════════════════════╪══════════════════════════════╪══════════════════════════════╣
║trigger.time-unit             │The TimeUnit to apply to delay│<none>                        │java.util.concurrent.TimeUnit ║
║                              │values.                       │                              │                              ║
║trigger.fixed-delay           │Fixed delay for periodic      │1                             │java.lang.Integer             ║
║                              │triggers.                     │                              │                              ║
║trigger.cron                  │Cron expression value for the │<none>                        │java.lang.String              ║
║                              │Cron Trigger.                 │                              │                              ║
║trigger.initial-delay         │Initial delay for periodic    │0                             │java.lang.Integer             ║
║                              │triggers.                     │                              │                              ║
║trigger.max-messages          │Maximum messages per poll, -1 │1                             │java.lang.Long                ║
║                              │means infinity.               │                              │                              ║
║trigger.date-format           │Format for the date value.    │<none>                        │java.lang.String              ║
║jdbc.max-rows-per-poll        │Max numbers of rows to process│0                             │java.lang.Integer             ║
║                              │for each poll.                │                              │                              ║
║jdbc.query                    │The query to use to select    │<none>                        │java.lang.String              ║
║                              │data.                         │                              │                              ║
║jdbc.update                   │An SQL update statement to    │<none>                        │java.lang.String              ║
║                              │execute for marking polled    │                              │                              ║
║                              │messages as 'seen'.           │                              │                              ║
║jdbc.split                    │Whether to split the SQL      │true                          │java.lang.Boolean             ║
║                              │result as individual messages.│                              │                              ║
╚══════════════════════════════╧══════════════════════════════╧══════════════════════════════╧══════════════════════════════╝

dataflow:>
```
I expect to see the `DataSourceProperties` in there but they are missing.